src/presetsController.ts

```diff
@@ -1136,7 +1136,7 @@ export class PresetsController implements vscode.Disposable {
 
         preset.expandConfigurePresetForPresets(this.folderPath, 'build');
 
-        const allPresets = preset.buildPresets(this.folderPath).concat(preset.userBuildPresets(this.folderPath));
+        const allPresets = await this.getAllBuildPresets();
```

Comment:
Is there a reason these methods weren't used before here (since I see they aren't new)?

Reply:
I think this was a mishap, where we should've switched to using this method before at some point, but didn't

Reply:
ah gotcha